templates/frontend/pages/submissions.tpl

```diff
@@ -21,6 +21,9 @@
 	<div class="cmp_notification">
 		{if $sections|@count == 0}
 			{translate key="author.submit.notAccepting"}
+		{elseif !$currentContext->getData('enableSubmissions')}
+			{capture assign=url}{url page="management" op="settings" path="workflow" anchor="submission/allowSubmissions"}{/capture}
+			{translate key="manager.setup.allowSubmissions.notAccepting"}
```

Comment:
The locale key needs to be changed to `author.submit.notAccepting` here and everywhere this locale key is used. 

Also, the `url` captured here is never used. Ideally, the locale string will include a URL within it and this could be passed in. But if this locale key is used in a flying notification (those toast-style notifications that appear at the top right), then the link won't work. So include it or not based on your judgement.